base/base/Decimal.h

```diff
@@ -49,15 +49,15 @@ struct Decimal
     using NativeType = T;
 
     constexpr Decimal() = default;
-    constexpr Decimal(Decimal<T> &&) = default;
+    constexpr Decimal(Decimal<T> &&) noexcept = default;
```

Comment:
In theory, a =-defaulted ctor should already be `noexcept` when applicable. But of course making it explicit is also fine.

(And just as suggestions: The reader wonders a) if the copy ctors/ops could also be made noexcept (they could indeed) and b) why ctor `constexpr Decimal(const T & value_): value(value_) {}` is not defaulted.)